README.md

```diff
@@ -36,5 +36,27 @@ data = llsd.parse(data_binary)
 # >>> {'foo: 'bar'}
 ```
 
+## Develop
+
+Requirements:
+
+- [pre-commit](https://pre-commit.com/)
+
+Set up a venv and install development dependencies:
+```
+python3 -m venv .venv
+. .venv/bin/activate
+pip install .[dev]
+```
+
+Run tests:
+```
+pytest
+```
+
+### Benchmarks
+
+Benchmarks from commits to `main` are published [here](https://secondlife.github.io/python-llsd/dev/bench/).
```

Comment:
Could the check unconditionally produce a summary of perf changes even when they're not negative?

Reply:
There is supposed to be an option to produce job summary reports containing comparison data, https://github.com/benchmark-action/github-action-benchmark#pr-summary However, when I tried to enable this the action failed due to a git operation failing. This seems to be related to an open issue, https://github.com/benchmark-action/github-action-benchmark/issues/158

Reply:
There seems to be quite a bit of recent activity, and that issue is comparatively new. Once fixed/addressed we can try to re-enable `summary-always: true`